test/datalog_tests/simple3.dl

```diff
@@ -0,0 +1,208 @@
+/******************************************
```

Comment:
This is just a test or is this supposed to be something we want to reuse?

Reply:
Just a test